search.go

```diff
@@ -3,10 +3,9 @@ package ldap
 import (
 	"errors"
 	"fmt"
+	ber "github.com/go-asn1-ber/asn1-ber"
```

Comment:
Minor nit, keep third party imports as a second import group, rather than mixed in with the stdlib imports.